CHANGES.md

```diff
@@ -2,6 +2,8 @@
 - [BREAKING CHANGE] Moved code for encryption on Android to the
   `sync-android-encryption` subproject. See the instructions in the [README](https://github.com/cloudant/sync-android/blob/master/README.md)
   file for how to include `sync-android-encryption` in your project.
+- [BREAKING CHANGE] Removed Configuration options from CouchConfig which are not used
```

Comment:
Configuration and CouchConfig should be in backticks to make it clearer.

'see commit' should either be in its own sentence or in brackets